storage/migrations/19_remove_unused_tables.go

```diff
@@ -4,11 +4,10 @@ import (
 	"github.com/go-pg/migrations/v8"
 )
 
-// Schema version 18 drops unused tables.
+// Schema version 19 drops unused tables.
 
 func init() {
 	up := batch(`
-	DROP MATERIALIZED VIEW IF EXISTS public.chain_visualizer_chain_data_view; 		-- artifact from chainwatch
```

Comment:
This view is being used by an external project and is removed in the hotfix's v17 migration. Removing this to prevent an error here.

Reply:
This view depends on some of the tables being dropped below:
https://github.com/filecoin-project/sentinel-visor/blob/master/storage/migrations/7_chainvis_views.go#L35
https://github.com/filecoin-project/sentinel-visor/blob/master/storage/migrations/7_chainvis_views.go#L37

Reply:
The view supports [this tool](https://github.com/DigitalMOB2/filecoin-lotus-explorer-playground/). If we are dubious about its value, I'd like to talk that through first, but maybe not as part of this PR.

Agree that the view needs to be corrected for the dropped tables. Will fix that.

Reply:
An alternative to fixing it here could be updating the view here: https://github.com/DigitalMOB2/filecoin-lotus-explorer-playground/blob/master/2.create-views.sql#L1. But happy to go with either option.